pkg/parser/parser.go

```diff
@@ -257,11 +275,13 @@ func (p *Parser) Parse(ctx context.Context, config string) (result *Result, err
 	}
 
 	// Create service tasks
-	serviceTasks, err := p.createServiceTasks(protoTasks)
-	if err != nil {
-		return nil, err
+	if p.noServiceTasks {
```

Comment:
Should it be `!p.noServiceTasks`? 🤔

Reply:
Fixed in https://github.com/cirruslabs/cirrus-cli/pull/454/commits/d529e4d497f08a271b6aaeddd907cb28580e86ca.